pegasus/test/test_hoc_routes.rb

```diff
@@ -105,6 +105,33 @@ class HocRoutesTest < Minitest::Test
       end
     end
 
+    it 'starts and ends given tutorial, tracking company and tutorial' do
```

Comment:
as long as the test is getting run, it's good enough for now. we should rename one of the tests next week though.